tests/util/registry/README.md

```diff
@@ -1,14 +1,14 @@
-# Using a Local Docker Registry
+# Using an in-cluster Docker Registry
```

Comment:
Let's move all of this to something like `/pkg/test/docker/registry`. The idea is that this is a library for testing.

Reply:
will do